third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt

```diff
@@ -1,5 +1,6 @@
 Tests that audits panel works.
 
+error: Inspected target navigated or closed
```

Comment:
it's a bit surprising to see these error states in a "successful" run. do you know whats going on here?

Reply:
this pr aint ready look away 👀